engine/apps/slack/models/slack_user_identity.py

```diff
@@ -132,9 +132,12 @@ def send_link_to_slack_message(self, slack_message: "SlackMessage"):
                 "elements": [
                     {
                         "type": "mrkdwn",
+                        # TODO: once _channel_id has been fully migrated to channel, remove _channel_id
+                        # see https://raintank-corp.slack.com/archives/C06K1MQ07GS/p173255546
+                        # f"<#{slack_message.channel.slack_id}>.\n"
                         "text": (
                             f"You received this message because you're not a member of "
-                            f"<#{slack_message.channel.slack_id}>.\n"
+                            f"<#{slack_message._channel_id}>.\n"
```

Comment:
"fix" from https://github.com/grafana/oncall/pull/5292 (let's wait to read from `channel` until the data-migration mentioned in that PR has been fully finished)